pyccel/codegen/printing/ccode.py

```diff
@@ -944,6 +946,22 @@ def _print_Import(self, expr):
             source = source.name[-1]
         else:
             source = self._print(source)
+        if expr.target:
+           dtype = expr.target.pop().name
+           if source.startswith('stc'):
+             dtype_macro = dtype.upper()
+             _,container_type = source.split("/")
+             class_type_macro = "SET" if container_type.startswith("hset") else "VEC"
```

Comment:
I think a dictionary would be appropriate for this as there will be at least 2 more cases added later (dicts and arrays)

Reply:
Fixed : 62e7694

Reply:
Yes, but I think `import_file,_ = source.split('_')` needs to come first

Reply:
Like this : cad722d ?

Reply:
Almost, if you test the code you should see what I mean

Reply:
> Yes, but I think `import_file,_ = source.split('_')` needs to come first

Could you please explain to me why we need this change?

Reply:
Fixed : 01150c5

Reply:
@Farouk-Echaref did you put this back?

Reply:
@EmilyBourne  no I don't think so, is there a version issue ? Ive only merged with the devel branch and added two more lines from my previous branch

Reply:
It seems you put it back in this commit: 0f58e7cde2689f8f4587b8cfdcd3129077bcc2a1
You should be careful when merging changes. I guess this was flagged as a merge conflict and was wrongly corrected. I don't see why it would reappear otherwise.
Ideally you should pull all changes before committing anything to be sure that you are adding to the newest version but if you have forgotten to do this you can get these changes. Personally I use `git pull --rebase` to do this as this preserves the order of the commits which often makes it easier to find the code that you added but I think git merges by default.